packages/runtime/container-runtime/src/deltaManagerProxies.ts

```diff
@@ -25,6 +25,61 @@ import {
 import type { PendingStateManager } from "./pendingStateManager.js";
 import { summarizerClientType } from "./summary/index.js";
 
+/**
+ * Interface defining event handlers for the BaseDeltaManagerProxy.
+ * These handlers allow customization of behavior for various DeltaManager events.
+ */
+export interface BaseDeltaManagerProxyEventHandlers {
```

Comment:
Great

Reply:
At the risk of nerd-sniping you... Can we tie this type to `IDeltaManagerEvents`?  Maybe later :)